participant/app/routes/about.tsx

```diff
@@ -0,0 +1,33 @@
+import { Alert } from "@trussworks/react-uswds";
+import { Button } from "@trussworks/react-uswds";
+import React from "react";
+import { Trans, useTranslation } from "react-i18next";
+
+export default function Index() {
+  const { t } = useTranslation();
+  const listProcessKeys: string[] = ["answer", "upload", "appointment"];
+
+  return (
+    <div className="measure-6">
```

Comment:
@makaelastephens Is this the measure we want on all the pages? If so, we might want to put it in the Layout

Reply:
Yeah I think we should use 6 as the base. @microwavenby do you want me to make a ticket for updating the layouts? Or is it simple to add that to this PR?